### DISCUSSION TITLE
Linear index to matrix index

### DISCUSSION THREAD

**Discussion by m-khansari** [mathnet/mathnet-numerics#830] [Q&A] [answered]
Hello
I have a set of linear indices for a matrix which needs to be set to zero, how can I convert them to matrix indices in math numerics 
is there a function in math numerics that can do that?

**Comment by cdrnet** [comment, marked as answer]
Linear indexing as e.g. in Matlab is very storage format specific. But it so happens that dense matrices in Numerics are organized the same way in column-major. So, if the matrix is dense, then `m.AsColumnMajorArray()` will return the internal array (directly, not a copy) which you can then access in this linear way.
